docs/devel/roadmap.md

```diff
@@ -0,0 +1,57 @@
+|View cluster resources       |Handle Pods - list and basic details view|P0|
+|View cluster resources       |Handle Deployments - list and basic details view|P0|
+|View cluster resources       |Handle Daemon Sets - list and basic details view|P0|
+|View cluster resources       |Handle Pet Sets - list and basic details view|P0|
+|View cluster resources       |UI is aware of namespaces/tenants|P0|
+|View cluster resources       |Handle Nodes - list and basic details view|P1|
+|View cluster resources       |Handle Jobs - list and basic details view|P1|
+|View cluster resources       |Show events for resources|P1|
+|Monitoring/troubleshooting   |Sparklines/metrics/graphs for Nodes|P1|
+|Monitoring/troubleshooting   |Sparklines/metrics/graphs for Containers|P1|
+|Monitoring/troubleshooting   |Sparklines/metrics/graphs for Pods|P1|
+|Monitoring/troubleshooting   |Surface errors/warnings for all resources shown|P1|
+|Manage apps/resources        |YAML edit for resources displayed|P1|
+|Manage apps/resources        |Delete support for resources displayed|P1|
+|Create apps                  |Deploy form: migrate to Deployment objects|P1|
+|Multi cluster (Ubernetes)    |Support multiple clusters in one UI|P1|
+|Security/IAM                 |Make the UI securely accessible from outside of a cluster|P1|
+|Security/IAM                 |Make the UI act on behalf of a user|P1|
+|Security/IAM                 |IAM: don't crash/work correctly when IAM is enabled|P1|
+|Nonfunctional                |Analytics|P1|
+|Nonfunctional                |"Send feedback"/"Report bug" capability|P1|
+|View cluster resources       |Show interconnections between resources: related resources, parents, children|P2|
+|Create apps                  |Deploy form: autosuggest from a container registry (GCR, docker hub, etc.)|P2|
+|Create apps                  |Deploy form: implement missing options (e.g., volumes, etc.)|P2|
+|View cluster resources       |Show logs for controller resources (e.g., logs for all Pods of a Replica Set)|P2|
+|View cluster resources       |Handle Containers - list and basic details view|P2|
+|View cluster resources       |Handle Secrets - list and basic details view|P2|
+|View cluster resources       |Handle Namespaces - list and basic details view|P2|
+|View cluster resources       |Handle Ingress - list and basic details view|P2|
+|Monitoring/troubleshooting   |Sparklines/metrics/graphs for Services|P2|
+|Monitoring/troubleshooting   |Logs viewer: show logs for O(days) time period|P2|
+|Monitoring/troubleshooting   |Logs viewer: handle large log files - O(GiB) in size|P2|
+|Monitoring/troubleshooting   |Events viewer: handle large logs streams - O(1k) events|P2|
+|Multi cluster (Ubernetes)    |Native support of Ubernetes control plane: Dashboard can talk to it|P2|
+|Security/IAM                 |IAM: support IAM as first class citzen (e.g., disable a button when cannot delete a resource)|P2|
+|Nonfunctional                |Internationalization|P2|
```

Comment:
Japanese speak english very poorly. Must be implemented in Kubernetes 1.3 timeframe. I give it a P1. Almost done anyway.